### PR TITLE
Changing the correction framwork and Tedner task to load 1D BC map

### PR DESCRIPTION
**Pull request by hahassan7**
The correction framework was changed so that it loads one dimensional Bad Channel Map.
In order to use it you have to set the flag "load1DBadChMap" to true in the yaml configuration file.

### CODE REVIEW

File: PWG/EMCAL/EMCALtasks/AliEmcalCorrectionComponent.h [hunk 1]
```diff
@@ -112,6 +112,7 @@ class AliEmcalCorrectionComponent : public TNamed {
  protected:
   PWG::Tools::AliYAMLConfiguration fYAMLConfig;           ///< Contains the %YAML configuration used to configure the component
   Bool_t                  fCreateHisto;                   ///< Flag to make some basic histograms
+  Bool_t                  fLoad1DBadChMap;                ///< Flag to load 1D bad channel map
```
Comment (raymondEhlers) [inline PWG/EMCAL/EMCALtasks/AliEmcalCorrectionComponent.h]: Please bump the ClassDef

**Comment by raymondEhlers** [comment]
+1 thanks!

**Comment by ktf** [comment]
+test

**Comment by ktf** [comment]
+1

**Comment by ktf** [comment]
@raymondEhlers apparently your "thanks" prevented the bot to understand the +1.

**Comment by raymondEhlers** [comment]
@ktf Okay, thanks for the heads up. To avoid issues in the future - did something change? Because I think I've said similar things in the past. Did it have something to do with needing additional approval from other contributors?

**Pull request merged**